plug-ins/groups/class_samurai.cpp

```diff
@@ -79,7 +80,7 @@ SKILL_RUNP( enchant )
 
     wear_level = get_wear_level( ch, obj );
 
-    if (wear_level > ch->getRealLevel( ))
+    if (wear_level > ch->getModifyLevel( ))
```

Comment:
@fiorine all checks for wear_level need to be done against `ch->getRealLevel()`. Remort level bonuses are already taken into account inside `get_wear_level` and don't need to be considered again.

Reply:
@ruffinakoza Fixed.